CHANGELOG.md

```diff
@@ -289,9 +299,19 @@ for info on changes for earlier releases.
 [`no-mutable-exports`]: ./docs/rules/no-mutable-exports.md
 [`prefer-default-export`]: ./docs/rules/prefer-default-export.md
 [`no-restricted-paths`]: ./docs/rules/no-restricted-paths.md
-
+<<<<<<< HEAD
+[`no-absolute-path`]: ./docs/rules/no-absolute-path.md
+=======
+[`max-dependencies`]: ./docs/rules/max-dependencies.md
+>>>>>>> tizmagik/feature/max-dependencies
```

Comment:
Ahah! My publish reviews are finally useful: There's are traces of merge conflicts here :smile: